src/Google.Maps/Internal/MapsHttp.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Newtonsoft.Json;
+
+namespace Google.Maps.Internal
+{
+	public class MapsHttp
+	{
+		JsonSerializerSettings settings = new JsonSerializerSettings
+		{
+			Converters = new List<JsonConverter> { new JsonEnumTypeConverter(), new JsonLocationConverter() }
+		};
+
+		public async Task<T> GetAsync<T>(Uri uri) where T : class
+		{
+			uri = SignUri(uri);
+
+			using (var client = new HttpClient())
```

Comment:
although HttpClient do implement IDisposable, all that I have read seem to indicate that should not be used in this manner.

This SO article is a great starting point. https://stackoverflow.com/questions/15705092/do-httpclient-and-httpclienthandler-have-to-be-disposed

I think it can thus be declared as a class level member and re-used.